README.md

```diff
@@ -311,6 +311,7 @@ For information about options that've changed, there's always [the changelog](ht
  - `proxy`       : Forwards request through HTTP(s) proxy. Eg. `proxy: 'http://user:pass@proxy.server.com:3128'`. For more advanced proxying/tunneling use a custom `agent`, as described below.
  - `headers`     : Object containing custom HTTP headers for request. Overrides defaults described below.
  - `case_sensitive_headers`     : Set to `true` to prevent header key lower casing. Defaults to `false`.
+ - `follow_authorization_header` : Set to `true` to retain authorization header when a redirect happens to a different hostname. Defaults to `false`.
```

Comment:
hmm, i thought the current behaviour was to retain the headers? maybe it would make more sense to have the existing behaviour as the default?